### ISSUE TITLE
Hierarcy slicher accessibility

### ISSUE THREAD

**Issue by elijoho** [liprec/powerbi-hierarchySlicer#96]
The regular slicer in Power BI have support for klicking CTRL + Right arrow to "enter" the filter. Therby you can control the filter using only keyboard. 
This does not work for hierarcy slicer and it is therefore not possible to use this awesome visual when creating reports that need to meet accessibility standards. 

---
#### Document details

⚠ Do not edit this section. It is required for docs.jppp.org ➟ GitHub issue linking.

* document_id=487cd175-be13-40df-a80e-aa1c7f531eb1
* document_url=https://azurebi-docs.jppp.org/powerbi-visuals/hierarchy-slicer.html

**Comment by liprec** [comment]
Thanks for the feedback and I will start implementing it for a new version.

(Moved the issue to the slicer repository)

**Comment by ckovvuri** [comment]
Thank you for implementing the fix. Is there any possible timeline\release version marked for this implementation. would like to keep track as the fix helps keyboard and screen reader users.

**Comment by cdcdatascientist** [comment]
I'd also love to bump this - do you have any timeline @liprec ?

**Comment by tnallen** [comment]
Hello, I am also interested in this feature. Is there a timeline when it will become available?

**Comment by cdcdatascientist** [comment]
@liprec - It looks like microsoft has turned on keyboard focus for custom visuals (https://docs.microsoft.com/en-us/power-bi/developer/visuals/supportskeyboardfocus-feature) ! 

Does that impact your timeline on this upate?

**Comment by liprec** [comment]
Sorry for you al, but the current support for keyboard is limited for custom visuals: the keystrokes are not exposed to the code and therefor I cannot identify if someone is pressing up or down or any other key.

**Comment by cdcdatascientist** [comment]
@liprec have you heard anything back from MS about this?
